applications/tests/test_reservations_utils.py

```diff
@@ -18,8 +18,11 @@
 
 @freeze_time("2019-01-14T08:00:00Z")
 @pytest.mark.parametrize("berth_switch", [True, False])
+@pytest.mark.parametrize("customer_private", [True, False])
```

Comment:
you could put all of the fixtures in a single decorator, something like:
```python
@pytest.mark.parametrize("berth_switch, customer_private, berth_switch_info", (
    [True, False], [True, False], [True, False])
)
```

Reply:
ah, yeah, and that indirect could be added too

Reply:
IMO it looks a bit clearer in different decorators.

Also, it's the way the [documentation](https://docs.pytest.org/en/latest/parametrize.html) suggests.
<img width="788" alt="image" src="https://user-images.githubusercontent.com/15201480/73913218-22eafb80-48bf-11ea-8500-5ced354135f0.png">

Reply:
well, the docs are a bit ambiguous 🙂 :
![image](https://user-images.githubusercontent.com/24206160/73914564-35b2ff80-48c2-11ea-8bc4-4aebbd6831bf.png)
but, yeah, it does look cleaner somewhat. aight, let'se keep it.

Reply:
Hmmm... strange, but ok. 👍